compiler/test/fail_compilation/unpacking.d

```diff
@@ -30,4 +31,6 @@ void main()
 
     extern (c, d) = tuple(1,2); // error
     auto extern (e, f) = tuple(1,2); // error
```

Comment:
It seems the extern lines are not errors ;-)

Reply:
Well, they are, not sure why DMD does not print them in this case. E.g., try this:

```d
import std.typecons;

void main(){
    extern (c, d) = tuple(1,2); // error
    auto extern (e, f) = tuple(1,2); // error
}
```

Reply:
Ah right, I guess they are semantic errors while you are testing for parse errors, maybe I should put them in a separate fail-compilation file.